packages/captp/lib/captp.js

```diff
@@ -10,31 +10,51 @@ export { E, HandledPromise, Nat, harden };
 
 export function makeCapTP(ourId, send, bootstrapObj = undefined) {
   let unplug = false;
+
+  // convertValToSlot and convertSlotToVal both perform side effects,
+  // populating the c-lists (imports/exports/questions/answers) upon
+  // marshalling/unmarshalling.  As we traverse the datastructure representing
+  // the message, we discover what we need to import/export and send relevant
+  // messages across the wire.
   const { serialize, unserialize } = makeMarshal(
     // eslint-disable-next-line no-use-before-define
     convertValToSlot,
     // eslint-disable-next-line no-use-before-define
     convertSlotToVal,
   );
 
+  // Used to construct slot names for promises/non-promises.
+  // In this verison of CapTP we use strings for export/import slot names.
+  // prefixed with 'p' if promises and 'o' otherwise;
   let lastPromiseID = 0;
   let lastExportID = 0;
+  // Since we decide the numbers for questions, we use this to increment
+  // the question key
   let lastQuestionID = 0;
 
-  const valToSlot = new WeakMap();
-  const slotToVal = new Map(); // exports
-  const questions = new Map(); // chosen by us
-  const answers = new Map(); // chosen by our peer
-  const imports = new Map(); // chosen by our peer
+  const valToSlot = new WeakMap();   // exports looked up by val
```

Comment:
These lint complaints are due to prettier only allowing a single space between the end of line and `//`.  Could you please fix this?